### ISSUE TITLE
Having issue while running the program.

### ISSUE THREAD

**Issue by JafferWilson** [vagrawal/deepsphinx#3]
Hello Vishal,
I have the following issue while running the program and I am not able to resolve it. Please help me:  
```
>>> from deepsphinx.api import Predict
>>> ds = Predict(Predict.default_flags(),'batch-21937.data-00000-of-00001')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/aims-whm/deepsphinx-master/deepsphinx/api.py", line 24, in __init__
    1.0)
  File "/home/aims-whm/deepsphinx-master/deepsphinx/seq2seq_model.py", line 241, in seq2seq_model
    keep_prob)
  File "/home/aims-whm/deepsphinx-master/deepsphinx/seq2seq_model.py", line 51, in encoding_layer
    dtype=tf.float32)
  File "/usr/local/lib/python3.5/dist-packages/tensorflow/python/ops/rnn.py", line 396, in bidirectional_dynamic_rnn
    seq_dim=time_dim, batch_dim=batch_dim)
  File "/usr/local/lib/python3.5/dist-packages/tensorflow/python/ops/rnn.py", line 389, in _reverse
    seq_dim=seq_dim, batch_dim=batch_dim)
  File "/usr/local/lib/python3.5/dist-packages/tensorflow/python/ops/array_ops.py", line 2355, in reverse_sequence
    name=name)
  File "/usr/local/lib/python3.5/dist-packages/tensorflow/python/ops/gen_array_ops.py", line 2633, in reverse_sequence
    batch_dim=batch_dim, name=name)
  File "/usr/local/lib/python3.5/dist-packages/tensorflow/python/framework/op_def_library.py", line 589, in apply_op
    param_name=input_name)
  File "/usr/local/lib/python3.5/dist-packages/tensorflow/python/framework/op_def_library.py", line 60, in _SatisfiesTypeConstraint
    ", ".join(dtypes.as_dtype(x).name for x in allowed_list)))
TypeError: Value passed to parameter 'seq_lengths' has DataType float64 not in list of allowed values: int32, int64

```

**Comment by JafferWilson** [comment]
I have seen on the line number 51, there is not written int32. I found float32, which is correct. Still I do not understand from where this int32 is emerging?

**Comment by vagrawal** [comment]
Thanks for reporting this. This only occurs in python 3. While I absolutely aim to support python 3, I have used python 2 for testing this, so there might be some inconsistencies.

Also, as another note, I do not think the pre-trained model I provided will work well, as that is trained for comparing with the results of previous work, so I normalized by speaker. I think it will only work when the data is in the normalized format. If you want to use a pre-trained model, I request you to wait for few days, as I am planning to write instructions on how to use the API and train a model for the unnormalized speech data.
